shared/chat/conversation/info-panel/menu/index.tsx

```diff
@@ -128,14 +129,15 @@ class InfoPanelMenu extends React.Component<Props> {
 
     const isAdhoc = !!(props.convProps && props.convProps.teamType === 'adhoc')
```

Comment:
This seems to be broken; I'm getting the team menu on all my ad-hoc conversations too.

Reply:
This is an unrelated bug that hotpot has I think